cellprofiler/modules/measureimageareaoccupied.py

```diff
@@ -92,13 +95,10 @@ def __init__(self):
                         O_BINARY_IMAGE,
                         O_OBJECTS
                     ],
-                    doc="""
-                    The area can be measured in two ways:
-                    <ul>
-                        <li><i>{O_BINARY_IMAGE}:</i> The area occupied by the foreground in a binary (black and white)
-                        image.</li>
-                        <li><i>{O_OBJECTS}:</i> The area occupied by previously-identified objects.</li>
-                    </ul>
+                    doc="""\
+The area can be measured in two ways:
+-  *{O_BINARY_IMAGE}:* The area occupied by the foreground in a binary (black and white) image.
```

Comment:
Please put a newline between the previous sentence and the start of the bulleted list, otherwise the list does not render correctly.